eng/pipelines/templates/stages/archetype-spec-gen-sdk.yml

```diff
@@ -298,9 +318,31 @@ stages:
               -PRBranch "$(SdkPullRequestSourceBranch)"
               -AuthToken "$(azuresdk-github-pat)"
               -PRTitle "$(PrTitle)-generated-from-$(Build.DefinitionName)-$(Build.BuildId)"
-              -PRBody "$(GeneratedSDKInformation)"
+              -PRBody "$(GeneratedSDKInformation)$(ReleasePlanInfo)"
```

Comment:
IF ReleasePlanInfo is not set anywhere this will give the literal string `$(ReleasePlanInfo)`